frontend/src/pages/facemash.js

```diff
@@ -11,11 +11,10 @@ import Candidate from '../components/candidateBox'
 import { Container } from 'reactstrap'
 type Props = {}
```

Comment:
let's define the types of the props here. https://flow.org/en/docs/types/

doesn't have to be too specific, can just be type Object or something like that. This will improve readability, especially when we come back to this in later semesters